pkg/services/folder/folderimpl/folder.go

```diff
@@ -485,7 +516,9 @@ func (s *Service) Move(ctx context.Context, cmd *folder.MoveFolderCommand) (*fol
 	}
 
 	return s.store.Update(ctx, folder.UpdateFolderCommand{
```

Comment:
since in the update we check the signinuser for update and keep userid which update the folder record, should we pass the signinuser here in the command,